### PR TITLE
We weren't showing the progress stripe correctly - peer reviewablilit…

### PR DESCRIPTION
**Pull request by mehalshah**
…y is "false" or "true" so levels that were marked as "false" were being considered peer reviewable. Also I kind of hate untyped languages right now.

Note that this is diffed against staging. Open question on if this meets the bar for merging to staging.

### CODE REVIEW

File: dashboard/app/models/plc/enrollment_module_assignment.rb [hunk 1]
```diff
@@ -39,12 +39,12 @@ def status
...
   def self.stages_based_status(stages, user, script)
     all_levels = stages.flat_map(&:script_levels).flat_map(&:levels)
-    levels_tracked = all_levels.reject {|level| [External, ExternalLink].include?(level.class) || level.try(:peer_reviewable) }
+    levels_tracked = all_levels.reject {|level| [External, ExternalLink].include?(level.class) || level.try(:peer_reviewable).try(:to_bool) }
 
     user_progress_on_tracked_levels = UserLevel.where(user: user, level: levels_tracked, script: script)
     passed_levels = user_progress_on_tracked_levels.passing
 
-    peer_reviewable_levels = all_levels.select {|level| level.try(:peer_reviewable)}
+    peer_reviewable_levels = all_levels.select {|level| level.try(:peer_reviewable).try(:to_bool)}
```
Comment (aoby) [inline dashboard/app/models/plc/enrollment_module_assignment.rb]: Can you override this property in `Level` to always return bool, so the `to_bool` is only in one place?

Comment (aoby) [reply dashboard/app/models/plc/enrollment_module_assignment.rb]: For instance, you could add this to `free_response.rb`:
```ruby
  def peer_reviewable?
    self.peer_reviewable.try(:to_bool)
  end
```

and then here, just check `level.try(:peer_reviewable?)`

Comment (mehalshah) [reply dashboard/app/models/plc/enrollment_module_assignment.rb]: I'll do that

File: dashboard/app/models/free_response.rb [hunk 2]
```diff
@@ -44,4 +44,8 @@ class FreeResponse < Level
   def self.create_from_level_builder(params, level_params)
     create!(level_params.merge(user: params[:user], game: Game.free_response, level_num: 'custom'))
   end
+
+  def peer_reviewable?
+    try(:peer_reviewable).to_bool
```
Comment (ashercodeorg) [inline dashboard/app/models/free_response.rb:49]: Given that `to_bool` is not a method on `nil`, I'm confused why we are `try`ing here and not returning a default if the result of the `try` is `nil`.

Comment (aoby) [reply dashboard/app/models/free_response.rb:49]: Good catch! Do we know if this is failing in the wild? Agreed, we should return a default in the `nil` case.

**Pull request merged**